DVR/SessionDataTask.swift

```diff
@@ -35,7 +35,12 @@ class SessionDataTask: NSURLSessionDataTask {
         // Find interaction
         if let interaction = cassette?.interactionForRequest(request) {
             // Forward completion
-            completion?(interaction.responseData, interaction.response, nil)
+            if let completion = completion {
+                let queue = dispatch_queue_create("com.venmo.DVR.sessionDataTaskQueue", nil)
```

Comment:
I thought NSURLSession's queue's were nondeterministic so I wanted to replicate that here. Performance really is not really too important here, since we're only using this in tests (and the drop in performance of creating a new queue on each request is probably insignificant)

That said, I'm open to creating a static queue.